src/pages/AboutUs/index.js

```diff
@@ -0,0 +1,59 @@
+import React, { useEffect } from 'react'
+import styles from './styles.module.css'
+import CardGithub from '../../components/Card/CardGithub';
+import CardDiscord from '../../components/Card/CardDiscord';
+import Contribuitors from '../../components/Contributors';
+import MobileCardDiscord from '../../components/Card/MobileCardDiscord'
+import MobileCardGithub from '../../components/Card/MobileCardGithub'
+import { useTranslation } from 'react-i18next';
+
+export default function AboutUs() {
+
+    const { t } = useTranslation();
+    
+    return (
+        <>
+        <section className={styles.aboutUs}>
+            <section className={styles.firstStepAboutUS}>
+                <div className={styles.firstStepContentWrapper}>
+                    <h2>{t("about.mainTitle")}</h2>
+                    <p>{ t('about.firstPartDescription')}</p>
+                    <p>
+                        { t('about.secondPartDescription')}
+                        <strong>{ t('about.openDevName')}</strong>
+                        { t('about.thirdPartDescription')}
```

Comment:
Descobri que o i18-next já possui uma solução pra isso (a mesma coisa pra linha 35)
https://stackoverflow.com/questions/65975098/i18next-bold-text-in-the-middle-of-a-translation

```suggestion
                       <Trans i18nKey="about.secondPartDescription" />
```